src/controller.cpp

```diff
@@ -172,14 +158,10 @@ TricycleSteeringCmdVel Controller::computeTricycleModelInverseKinematics(
 }
 
 void Controller::setPlan(
-  const geometry_msgs::Transform & current_tf, const geometry_msgs::Twist & odom_twist,
-  const std::vector<geometry_msgs::PoseStamped> & global_plan)
+  const tf2::Transform & current_tf, const geometry_msgs::Twist & odom_twist,
+  const std::vector<tf2::Transform> & global_plan)
 {
-  ROS_DEBUG("TrackingPidLocalPlanner::setPlan(%d)", (int)global_plan.size());
-  ROS_DEBUG("Plan is defined in frame '%s'", global_plan.at(0).header.frame_id.c_str());
```

Comment:
Where did this log message go?

Reply:
I removed it. Since it was only debug logging, it didn't seem that important. The information to log this is no longer available (here).

If you want I can move the logging to `TrackingPidLocalPlanner`.

Reply:
Ah yes because Transforms don't contain have a header